src/Sulu/Bundle/ContactBundle/Entity/CountryRepository.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContactBundle\Entity;
+
+use Symfony\Component\Intl\Intl;
+
+/**
+ * Country repository.
+ */
+class CountryRepository
+{
+    public function findAll(string $displayLocale = null)
+    {
+        $countries = [];
+
+        foreach (Intl::getRegionBundle()->getCountryNames($displayLocale) as $code => $countryName) {
+            $countries[$code] = [
+                'code' => $code,
+                'name' => $countryName,
```

Comment:
I am not a big fan of passing untyped associative arrays around :confused: Do we need this repository at all? Another way to go about this is to simply use the `Intl` stuff directly. Otherwise I would also see value in introducing some kind of `Country` model object (not an entity!), which is not saved in the DB.

Reply:
As said above the repository was more to keep the old UI running. Will deprecate it if its ok?